### DISCUSSION TITLE
제페토 앱 테스트에서의 디버깅과 유니티 에디터와의 차이

### DISCUSSION THREAD

**Discussion by lhj5709** [naverz/zepeto-studio-kor#510] [5. Q&A]
상황을 설명드리면 2개의 씬이 있고 씬 이동 포탈을 구현한 상태입니다.

유니티 에디터에서는 정상적으로 이동 후 복귀가 가능하나 모바일 앱(QR생성) 에서 접속했을 때는 포탈이 전혀 작동을 안합니다.

1. 해당 기능 외에도 유니티 에디터에서는 잘 작동하는데 모바일에서는 작동이 안되는 것들이 있는데 어떤 부분을 확인해봐야 하나요?

2. 모바일 앱에서 디버그 로그를 확인할 수 없나요? 아니면 다른 디버깅 방법이 있나요 ?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ Scene을 이동할때 사용한 코드를 올려주시면 파악에 도움이 될 것 같습니다.
